cmd/main.go

```diff
@@ -312,7 +312,8 @@ func main() {
 	eventsHandler := createEventsHandler(crdEventsHandler, db, authzHandler, notificationStream, log)
 
 	prometheusRegistry := prometheus.DefaultRegisterer
-	metricsManager := metrics.NewMetricsManager(prometheusRegistry, eventsHandler)
+	directoryUsageMonitorConfig := metrics.NewDirectoryUsageCollectorConfig([]string{Options.WorkDir})
+	metricsManager := metrics.NewMetricsManager(prometheusRegistry, eventsHandler, metrics.NewOSDiskStatsHelper(), directoryUsageMonitorConfig, log)
```

Comment:
nit: not very important, however by passing `directoryUsageMonitorConfig` we're hinting to add one parameter each config that might be needed by collectors/metrics registered within the manager. I would've envisioned this as `MetricManagerOptions.DirectoryUsageMonitorConfig`